autonomy/chain/base.py

```diff
@@ -43,6 +43,17 @@ class UnitType(Enum):
     AGENT = 1
 
 
+class ServiceState(Enum):
+    """Service state."""
```

Comment:
Same as https://github.com/valory-xyz/autonolas-registries/blob/main/contracts/ServiceRegistry.sol#L41-L48

Reply:
Please add that info to the docstring, referencing a tag rather than main so it is static